modules/dankMaterialShell/meta.nix

```diff
@@ -0,0 +1,7 @@
+{ lib, ... }:
+{
+  name = "DankMaterialShell";
+  homepage = "https://github.com/AvengeMedia/DankMaterialShell";
+  maintainers = [ lib.maintainers.khas-amir ];
+  description = "Restart the panel to apply the theme";
```

Comment:
Could you rephrase the description with the IMPORTANT admonition to match the current style:

```console
$ rg --files-with-matches '> \[!IMPORTANT\]'
doc/src/modules.md
modules/blender/meta.nix
modules/discord/meta.nix
modules/firefox/meta.nix
modules/i3/meta.nix
modules/neovim/meta.nix
modules/spicetify/meta.nix
modules/sway/meta.nix
modules/zen-browser/meta.nix
```